src/APIFunctions/S3Bucket.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+export function uploadToS3(files) {
+  for (let i = 0; i < files.length; i++) {
+    let file = files[i];
+    let fileParts = files[i].name.split('.');
+    let fileName = fileParts[0];
+    let fileType = fileParts[1];
+    axios
+      .post('http://localhost:8082/api/S3Bucket/getSignedUrl', {
+        fileName: fileName,
```

Comment:
as mentioned in the comment before, lets just send a single param over `fileName` and handle the split over there.
```suggestion
        fileName,
```